setup.py

```diff
@@ -162,6 +174,8 @@ def run_setup(with_rdkafka=True):
     )
 
 try:
+    if not is_cpython:
+        raise ve_build_ext.UnsupportedPlatform(python_implementation)
```

Comment:
Would this `raise..except` pair make more sense as a simple boolean check on `is_cpython`? It seems like that would make the logic clearer.